demos/ssh-demo/k8s-cc-ssh.yaml

```diff
@@ -24,5 +24,5 @@ spec:
       runtimeClassName: kata
       containers:
       - name: ccv0-ssh
-        image: ${to_be_filled}
+        image: docker.io/katadocker/ccv0-ssh
```

Comment:
I guess it makes sense to hard-code this for a demo, but seeing this made me think of https://github.com/kata-containers/kata-containers/issues/3162 (aka you could keep that variable and just expand it using `envsubst(1)`).

Reply:
Hm. For the purposes of this demo and its target of maximum accessibility, I'd rather keep it as is.